src/BirdMessenger/TusBuild.cs

```diff
@@ -7,47 +7,50 @@
 
 namespace BirdMessenger
 {
-    public  class TusBuild
+    public sealed class DefaultTusBuild : TusBuild
     {
-        private TusClientOption _tusClientOptions = new TusClientOption();
-        
-        private bool _tusBuilt;
-        
-        public IDictionary<object, object> Properties { get; } = new Dictionary<object, object>(); 
-        
-        private List<Action<TusClientOption>> _configureClientActions= new List<Action<TusClientOption>>();
-
-        
+        private DefaultTusBuild()
+        {
+            _tusClientOptions = new DefaultTusClientOption();
+        }
 
-        public static TusBuild DefaultTusClientBuild(Uri tushost,string clientName="")
+        public DefaultTusClientOption Options => _tusClientOptions as DefaultTusClientOption;
+        public static DefaultTusBuild DefaultTusClientBuild(Uri tushost, string clientName = "")
```

Comment:
clientName is useless，will we remove it ？

Reply:
It would only be useful only in the case of non-default client builder. In default builder it makes no sense because you already provide the http clients.